Explorer/Assets/DCL/NameTags/NametagPlacementSystem.cs

```diff
@@ -73,25 +73,24 @@ private void AddTag([Data] in CameraComponent camera, Entity e, in AvatarShapeCo
             if (partitionComponent.IsBehind || IsOutOfRenderRange(camera, characterTransform)) return;
 
             NametagView nametagView = nametagViewPool.Get();
-            activeNametags.Add(avatarShape.ID, nametagView);
             nametagView.Id = avatarShape.ID;
             nametagView.Username.color = chatEntryConfiguration.GetNameColor(avatarShape.Name);
             nametagView.InjectConfiguration(chatBubbleConfigurationSo);
             nametagView.SetUsername($"{avatarShape.Name}<color=#76717E>#{avatarShape.ID}</color>");
             nametagView.gameObject.name = avatarShape.ID;
 
+
             UpdateTagPosition(nametagView, camera.Camera, characterTransform.Position);
 
             World.Add(e, nametagView);
         }
 
         [Query]
         [All(typeof(ChatBubbleComponent))]
-        private void ProcessChatBubbleComponents(Entity e, in ChatBubbleComponent chatBubbleComponent)
+        private void ProcessChatBubbleComponents(Entity e, in ChatBubbleComponent chatBubbleComponent, in NametagView nametagView)
```

Comment:
You should modify `ChatController` to place `ChatBubbleComponent` on the player entity, otherwise this query will never work